src/web/mjs/connectors/Kiryuu.mjs

```diff
@@ -8,10 +8,11 @@ export default class Kiryuu extends WordPressMangastream {
         super.label = 'Kiryuu';
         this.tags = [ 'manga', 'indonesian' ];
         this.url = 'https://kiryuu.co';
-        this.path = '/manga/?list';
+        this.path = '/manga/list-mode/';
 
-        this.queryChapters = 'div.bxcl ul li span.lchx a';
-        this.queryPages = 'div#readerarea > :not(.kln) source[src]:not([src=""])';
+        this.querMangaTitleFromURI = 'div#content div.postbody article h1.entry-title';
```

Comment:
Template uses `h1` instead of `h1.entry-title`, not sure if it is safe to add `h1.entry-title` to template, but i guess the `.entry-title` class can be skipped anyway